integration/qbft/scenarios/round_change.go

```diff
@@ -103,14 +103,14 @@ func roundChangeInstanceValidator(consensusData []byte, operatorID spectypes.Ope
 			Data: consensusData,
 		}).Encode()
 		if err != nil {
-			return fmt.Errorf("encode prepare data: %w", err)
+			panic(err)
```

Comment:
https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852

Reply:
it overwrites on future brunches